src/sections/Company/News-single/index.js

```diff
@@ -56,11 +57,19 @@ const NewsSingle = ({data}) => {
               <MDXRenderer>{body}</MDXRenderer>
             </SRLWrapper>
           </div>
-          <RelatedPosts 
+          <div style={{ display: "flex" }}>
+            <h5>
+              Read the full article on
+            </h5>
+            <a href={frontmatter.eurl} target="_blank" rel="noopener noreferrer">
```

Comment:
@Nikhil-Ladha @debo19 how to reference this eurl to make it functional? I've added it to the query...

Reply:
@Chadha93 add it to the query on `src/templates/news-single.js` as well